src/webgl/light.js

```diff
@@ -36,6 +36,30 @@ import p5 from '../core/main';
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ * let angle = 0;
+ * let light;
+ * function setup() {
+ *    createCanvas(100, 100, WEBGL);
+ *    slider = createSlider(0, 255, 255);
+ *    slider.position(width/2 - 40, height+5);
+ *    slider.style('width', '80px');
+ * }
+ * function draw() {
+ *    background(51);
+ *    light = slider.value();
```

Comment:
these reference examples should be as simple and concise as possible to reduce chance of confusion. 
1. i think we can simplify this by removing the slider. 
2. let's make two examples. the first without any rotation, just a static box. the second with the rotation.